datafusion/physical-plan/src/windows/mod.rs

```diff
@@ -217,6 +217,41 @@ fn get_casted_value(
     }
 }
 
+fn rewrite_null_expr_and_data_type(
+    args: &[Arc<dyn PhysicalExpr>],
+    expr_type: &DataType,
+) -> (Arc<dyn PhysicalExpr>, DataType) {
+    let expr = Arc::clone(&args[0]);
```

Comment:
lets assert args are non empty before access the first element

Reply:
Thanks. Fixed.